deps/Versions.make

```diff
@@ -20,7 +20,7 @@ GMP_BB_REL = 4
 MPFR_VER = 4.0.2
 MPFR_BB_REL = 2
 PATCHELF_VER = 0.9
-MBEDTLS_VER = 2.16.0
+MBEDTLS_VER = 2.16.6
 MBEDTLS_BB_REL = 1
```

Comment:
You'll need to change this `BB_REL` as well to whatever the build number it gets assigned is, then afterwards you should delete all the `MbedTLS` checksums and then run `./contrib/refresh_bb_tarballs.sh mbedtls` to get the new checksums.